components/kubeflow/kfserving/src/sample-pipeline.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import kfp.dsl as dsl
+from kfp import components
+import json
+
+kfserving_op = components.load_component_from_file('component.yaml')
```

Comment:
This sample pipeline doesn't need to be in src. We should put this file to the parent directory so that it can load the component.yaml.

Reply:
done